src/data/scripts/ai/VayraAntiFighterAI.java

```diff
@@ -23,193 +29,191 @@ public class VayraAntiFighterAI implements MissileAIPlugin, GuidedMissileAI {
-     *
-     *  FULL_RANDOM,
+     * 
+     *  FULL_RANDOM, 
      * The missile will always seek a random valid threat within arc around itself.
-     *
+     * 
      *  IGNORE_SOURCE,
      * The missile will pick the closest target of interest. Useful for custom MIRVs.
-     *
-     */
-    private final MagicTargeting.targetSeeking seeking = MagicTargeting.targetSeeking.LOCAL_RANDOM;
-
+     * 
+    */
+    private final MagicTargeting.targetSeeking seeking = MagicTargeting.targetSeeking.LOCAL_RANDOM;  
+    
     //////////////////////
     //    VARIABLES     //
     //////////////////////
-
+    
     private final float MAX_SPEED; // max speed of the missile after modifiers.
     private final float OFFSET; // Random starting offset for the waving.
     private CombatEngineAPI engine;
     private final MissileAPI MISSILE;
     private CombatEntityAPI target;
     private Vector2f lead = new Vector2f();
-    private boolean launch = true;
-    private float timer = 0, check = 0f;
-
+    private boolean launch=true;
+    private float timer=0, check=0f;
+    
     public VayraAntiFighterAI(MissileAPI missile, ShipAPI launchingShip) {
         this.MISSILE = missile;
         MAX_SPEED = missile.getMaxSpeed();
-        if (missile.getSource().getVariant().getHullMods().contains("eccm")) {
-            ECCM = 1;
-        }
+        if (missile.getSource().getVariant().getHullMods().contains("eccm")){
+            ECCM=1;
+        }        
         //calculate the precision range factor
-        PRECISION_RANGE = (float) Math.pow((2 * PRECISION_RANGE), 2);
-        OFFSET = (float) (Math.random() * MathUtils.FPI * 2);
+        PRECISION_RANGE=(float)Math.pow((2*PRECISION_RANGE),2);
+        OFFSET=(float)(Math.random()*MathUtils.FPI*2);
     }
-
+    
     //////////////////////
     //   MAIN AI LOOP   //
     //////////////////////
-
+    
     @Override
     public void advance(float amount) {
-
+        
         if (engine != Global.getCombatEngine()) {
             this.engine = Global.getCombatEngine();
         }
-
+        
         //skip the AI if the game is paused, the missile is engineless or fading
-        if (Global.getCombatEngine().isPaused() || MISSILE.isFading() || MISSILE.isFizzling()) {
-            return;
-        }
-
+        if (Global.getCombatEngine().isPaused() || MISSILE.isFading() || MISSILE.isFizzling()) {return;}
+        
         //assigning a target if there is none or it got destroyed
         if (target == null
-                || (TARGET_SWITCH
-                && ((target instanceof ShipAPI && !((ShipAPI) target).isAlive())
-                || !engine.isEntityInPlay(target))
-        )
-        ) {
+                || (TARGET_SWITCH 
+                        && ((target instanceof ShipAPI && !((ShipAPI) target).isAlive())
+                                  || !engine.isEntityInPlay(target))
+                   )
+                ){
```

Comment:
please revert so they're lined up. yes i know autoformatter has a problem with this, it needs to be fixed manually.